README.md

```diff
@@ -43,14 +43,14 @@ Each Cloud Controller version maps to a specific ACA-Py version, which is outlin
 
 | Aries Cloud Controller Version | Aries Cloud Agent Python Version |
 | ------------------------------ | -------------------------------- |
-| 0.5.1-0.5.2                    | 0.7.3                            |
+| 0.5.3                          | 0.7.4                            |
```

Comment:
I think we can keep the old version in this table (as it is still released), and just add a new entry to it

Reply:
sure